mesheryctl/internal/cli/root/app/view.go

```diff
@@ -0,0 +1,129 @@
+package app
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/ghodss/yaml"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	viewAllFlag   bool
+	outFormatFlag string
+)
+
+var viewCmd = &cobra.Command{
+	Use:   "view <application name>",
+	Short: "Display application(s)",
+	Long:  `Displays the contents of a specific application based on name or id`,
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+		application := ""
+		isID := false
+		// if application name/id available
+		if len(args) > 0 {
+			if viewAllFlag {
+				return errors.New("-a cannot be used when [application-name|application-id] is specified")
+			}
+			application = args[0]
+			// check if the application argument is a valid uuid v4 string
+			isID, err = regexp.MatchString("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$", application)
```

Comment:
Do shift this to an appropriate place so that we check if there's only one arg and it's an ID, else we grab all the args and format a word.

Reply:
managed